### ISSUE TITLE
Simplify deployment weblets

### ISSUE THREAD

**Issue by rkhamis** [threefoldtech/grid_weblets#489]
user wouldn't have to know min requirments for solutions they deploy.

"large", "medium" and "small" options only

should be done for:
 - [ ] peertube
 - [ ] taiga
 - [ ] mattermost
 - [ ]

**Comment by MohamedElmdary** [comment]
@abom @rkhamis @xmonader Please provide the sm/md/lg standard for each of these solution

**Comment by sasha-astiadi** [comment]
@rkhamis whats the update now? any implementation started this week?

**Comment by rkhamis** [comment]
I agree it should be a solution-based approach. not every solution has the same requirements.

regarding customizable deployments, I think this should be in an "advanced" section. or by default we show the three sizes and then an option for "manually customize size" or something.
